portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/cpdocumentsandmedia/CPDocumentsandmedia.testcase

```diff
@@ -55,47 +55,10 @@
 		</execute>
 	</command>
 
-	<command name="AddDocument" priority="4">
```

Comment:
@brianwulbern ah, okay. it looks like LRQA-31707 was never actually merged onto master. it exists on 7.0.x / ee-7.0.x. we might need to cherry-pick LRQA-31707 in first.